raiden/src/update_expression/set.rs

```diff
@@ -126,7 +126,14 @@ impl<T: super::IntoAttrName> UpdateSetExpressionBuilder for SetExpressionFilledW
             SetValue::Attr(a) => {
                 let set_attr = a.into_attr_name();
                 let set_attr_name = format!("#{}", set_attr);
-                let expression = format!("{} = {}", attr_name, set_attr_name);
+                let expression = if self.if_not_exists {
+                    format!(
+                        "{} = if_not_exists({}, {})",
+                        attr_name, attr_name, set_attr_name
```

Comment:
For now it assumes the attribute to check presence is the same one as the attribute that's being set i.e. it only allows `SET price = if_not_exists(price, 1000)` and can't do something like `SET price = if_not_exists(unit_price, 1000)`